.cicd/test-package.run.sh

```diff
@@ -1,22 +1,28 @@
 #!/bin/bash
-set -eu
+set -euo pipefail
+
+. "${0%/*}/libfunctions.sh"
 
 echo '+++ :minidisc: Installing EOSIO'
 
-if [[ $(apt-get --version 2>/dev/null) ]]; then # debian family
-    UPDATE='apt-get update'
-    echo "$ $UPDATE"
-    eval $UPDATE
-    INSTALL="apt-get install -y /eos/*.deb"
-    echo "$ $INSTALL"
-    eval $INSTALL
-elif [[ $(yum --version 2>/dev/null) ]]; then # RHEL family
-    UPDATE='yum check-update || :'
-    echo "$ $UPDATE"
-    eval $UPDATE
-    INSTALL="yum install -y /eos/*.rpm"
-    echo "$ $INSTALL"
-    eval $INSTALL
+if [[ $(apt-get --version 2>/dev/null) ]]; then # debian family packaging
+    perform 'apt-get update'
+    perform 'apt-get install -y /eos/*.deb'
+elif [[ $(yum --version 2>/dev/null) ]]; then # RHEL family packaging
+    perform 'yum check-update || :'
+    perform 'yum install -y /eos/*.rpm'
+elif [[ $(brew --version 2>/dev/null) ]]; then # homebrew packaging
+    perform 'brew update'
+    perform 'mkdir homebrew-eosio'
+    perform 'git init homebrew-eosio'
+    perform 'cp *.rb homebrew-eosio'
+    perform "sed -i.bk -e 's/url \".*\"/url \"http:\/\/127.0.0.1:7800\"/' homebrew-eosio/*.rb"
+    perform "pushd homebrew-eosio && git add *.rb && git commit -m 'test it!' && popd"
```

Comment:
We don't need the ampersand chaining of commands (`&&`) outside of dockerfiles. I think it looks messy and is harder to follow than just making them individual lines. Especially if you're trying to see the output of one of the commands, or figure out which command threw an error message.